docs/Userguide_multinode.rst

```diff
@@ -0,0 +1,121 @@
+Multiple Nodes
+==============
+
+Data Parallel
+-------------
+
+.. image:: _static/dataparallel.png
+
+Request 3 nodes with each at least 4 GPUs each.
+
+.. code-block:: bash
+   :linenos:
+
+    # Number of Nodes
+    #SBATCH --nodes=3
+
+    # Number of tasks. 3 (1 per node)
+    #SBATCH --ntasks=3
+
+    # Number of GPU per node
+    #SBATCH --gres=gpu:4
+    #SBATCH --gpus-per-node=4
+
+    # 16 CPUs per node
+    #SBATCH --cpus-per-gpu=4
+
+    # 16Go per nodes (4Go per GPU)
+    #SBATCH --mem=16Go
+
+    # we need all nodes to be ready at the same time
+    #SBATCH --wait-all-nodes=1
+
+    # Total resources:
+    #   CPU: 16 * 3 = 48
+    #   RAM: 16 * 3 = 48 Go
+    #   GPU:  4 * 3 = 12
+
+    # Setup our rendez-vous point
+    RDV_ADDR=$(hostname)
+    WORLD_SIZE=$SLURM_JOB_NUM_NODES
+    # -----
+
+    srun -l torchrun \
+        --nproc_per_node=$SLURM_GPUS_PER_NODE\
+        --nnodes=$WORLD_SIZE\
+        --rdzv_id=$SLURM_JOB_ID\
+        --rdzv_backend=c10d\
+        --rdzv_endpoint=$RDV_ADDR\
+        training_script.py
+
+
+.. code-block:: python
+
+    import os
+    import torch.distributed as dist
```

Comment:
Maybe add a comment stating that this is the `training_script.py` mentioned in the slurm script above?

Reply:
added